examples/step-86/doc/results.dox

```diff
@@ -0,0 +1,243 @@
+the details of the time stepping algorithm. Recall that the solution above
+is controlled by the following parameters:
+@code
+  subsection Time stepper
+    subsection Running parameters
+      set final time              = 5
+      set initial step size       = 0.025
+      set initial time            = 0
+      set match final time        = false
+      set maximum number of steps = -1
+      set options prefix          =
+      set solver type             = beuler
+    end
+
+    subsection Error control
+      set absolute error tolerance = -1
+      set adaptor type             = none
+      set ignore algebraic lte     = true
+      set maximum step size        = -1
+      set minimum step size        = -1
+      set relative error tolerance = -1
+    end
+  end
+@endcode
+Of particular interest for us here is to set the time stepping algorithm
+and the adaptive time step control method. The latter is set to "none"
+above, but there are
+[several alternative choices](https://petsc.org/release/manualpages/TS/TSAdaptType/)
+for this parameter. For example, we can set parameters as follows,
+@code
+  subsection Time stepper
+    subsection Running parameters
+      set final time              = 5
+      set initial step size       = 0.025
+      set initial time            = 0
+      set match final time        = false
+      set maximum number of steps = -1
+      set options prefix          =
+      set solver type             = bdf
+    end
+
+    subsection Error control
+      set absolute error tolerance = 1e-2
+      set relative error tolerance = 1e-2
+      set adaptor type             = basic
+      set ignore algebraic lte     = true
+      set maximum step size        = 1
+      set minimum step size        = 0.01
+    end
+  end
+@endcode
+What we do here is set the initial time step size to 0.025, and choose relatively
+large absolute and relative error tolerances of 0.01 for the time step size
+adaptation algorithm (for which we choose "basic"). We ask PETSc TS to use a
+[Backward Differentiation Formula (BDF)](https://en.wikipedia.org/wiki/Backward_differentiation_formula)
+method, and we get the following as output:
+@code
+
+===========================================
+Number of active cells: 768
+Number of degrees of freedom: 833
+
+Time step 0 at t=0
+     5 linear iterations.
+     5 linear iterations.
+     5 linear iterations.
+     4 linear iterations.
+     6 linear iterations.
+Time step 1 at t=0.01
+     5 linear iterations.
+     5 linear iterations.
+Time step 2 at t=0.02
+     5 linear iterations.
+     5 linear iterations.
+Time step 3 at t=0.03
+     5 linear iterations.
+     5 linear iterations.
+Time step 4 at t=0.042574
+     5 linear iterations.
+     5 linear iterations.
+Time step 5 at t=0.0588392
+     5 linear iterations.
+     5 linear iterations.
+Time step 6 at t=0.0783573
+     5 linear iterations.
+     7 linear iterations.
+     5 linear iterations.
+     7 linear iterations.
```

Comment:
remesh? Maybe add a print in the callbacks?

Reply:
Yes, good idea!

(The remesh isn't happening at this place, but I assume you meant that as a comment in general.)